scripts/win-build.sh

```diff
@@ -15,4 +12,4 @@ pip install -v -e .
 # Build the wheel
 python setup.py bdist_wheel
 
-pyinstaller -y scripts/borg.exe.spec
+pyinstaller -y scripts/win-borg.exe.spec
```

Comment:
is it necessary to split borg.exe.spec into 2 files? if there's no strong reason to do that, I'ld prefer keeping it in one file (because some of the changes we need to do there will otherwise have to made twice, updating both files).

also, I don't see the win-borg.exe.spec in this PR, did you forget to git add it?

Reply:
it was git mv-ed because it was only used by the previous setup that included a general installation of msys2. 
Now it's done via the scoop installation and we don't all this clutter.

Reply:
the win-build.ps1 moved to win-build.sh. win- because to not confuse someone with this build and the .sh ending.